frontend/src/components/Home/Chart.css

```diff
@@ -53,75 +44,33 @@
 }
 
 .chart-content {
-    display: flex;
-    flex-direction: column; /* Organizăm graficele pe verticală */
-    gap: 30px; /* Spațiere între grafice */
-    overflow-y: auto; /* Scroll vertical dacă sunt prea multe grafice */
-    padding: 1rem;
+    display: grid; /* Layout de tip grilă */
+    grid-template-columns: 1; /* Ajustează numărul de coloane */
+    gap: 20px; /* Spațiere între grafice */
+    justify-items: center; /* Centrează fiecare grafic */
+    align-items: center; /* Centrează fiecare grafic vertical */
+    width: 100%;
+    margin-top: 20px; /* Spațiu între butoane și grafice */
 }
 
 .chart-item {
-    background-color: #ffffff; /* Fundal alb pentru grafice */
-    border: 1px solid #e1e1e1; /* Bordură subțire gri */
+    background-color: #ffffff;
+    border: 1px solid #e1e1e1;
     border-radius: 8px;
     padding: 20px;
     box-shadow: 0 4px 6px rgba(0, 0, 0, 0.1); /* Umbră subtilă */
+    width: 100%;
```

Comment:
Incearca sa fii consistent. Intr-o parte folosesti % in alta px. Ar trebui sa faci totul responsive din prima cu flex si flex box